spec/mercure.md

```diff
@@ -88,18 +81,20 @@ header). The target URL of these links **MUST** be a hub implementing the Mercur
 
 The publisher **MAY** provide the following target attributes in the Link Headers:
 
- *  `last-event-id`: the identifier of the last event dispatched by the publisher at the time of
+*   `last-event-id`: the identifier of the last event dispatched by the publisher at the time of
     the generation of this resource. If provided, it **MUST** be passed to the hub through a query
     parameter called `Last-Event-ID` and will be used to ensure that possible updates having been
     made between the resource generation by the server and the connection to the hub are not lost.
     See (#reconciliation).
-
- *  `content-type`: the content type of the updates that will be pushed by the hub. If omitted,
+*   `content-type`: the content type of the updates that will be pushed by the hub. If omitted,
     the subscriber **MUST** assume that the content type will be the same as that of the original
     resource. Setting the `content-type` attribute is especially useful to hint that partial updates
     will be pushed, using formats such as JSON Patch [@RFC6902] or JSON Merge Patch [@RFC7386].
-
- *  `key-set`: the URL of the key set to use to decrypt updates, encoded in the JWK set format
+*   `cookie-name`: the name of the cookie that contains the JWS [@!RFC7515] used for authorization
+    ((#authorization)). If this attribute is not set, the name of the cookie is assumed to be
+    `mercureAuthorization`. Using this attribute allows prevents cookie name conflicts when using
```

Comment:
> Using this attribute allows **to prevent** cookie name conflicts...

Reply:
Good catch. It should even be `preventing`.

Reply:
Yeah, was zig-zagging between the two, too... number of Google resuts for each phrase ([12.5M](https://www.google.com/search?q=%22allows+to+prevent%22) vs. [16.9k](https://www.google.com/search?q=%22allows+preventing%22)) and [this comparison](https://textranch.com/387138/allows-to-prevent/or/allows-preventing/) made me choose the "to prevent" version. :man_shrugging: